### ISSUE TITLE
'empty(...)' usage quick-fix error

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#1809]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended), 4.0.7.1 |
| Language level | PHP 7.4                     |

Current behaviour
--
```php
 if (empty($result_ids) === true) {
    return [];
}
```

⇒ 

```php
if (count($result_ids) === 0 === true) {
    return [];
}
```

Expected behaviour
--
```php
if (count($result_ids) === 0) {
    return [];
}
```

**Comment by ea-inspections-team** [comment]
Thank you for reporting @voku ! I am currently unable to reproduce the issue, which suggests it was resolved in a previous release. I will close this one as a duplicate.
